docs/releases/3.0.txt

```diff
@@ -473,6 +473,15 @@ Miscellaneous
 * ``intword`` template filter now translates ``1.0`` as a singular phrase and
   all other numeric values as plural. This may be incorrect for some languages.
 
+* The ``STATIC_URL`` and ``MEDIA_URL`` settings will now automatically have the
+  ``SCRIPT_NAME`` header prefixed to their values when the settings attribute
+  is requested. This auto-prefixation occurs only if *both* of the following are
+  true of the initial values: a) they are *not* valid URLs and b) they are pointing
+  to relative path. This may cause backwards incompatibility in certain rare corner
```

Comment:
The note about backwards incompatibility can be dropped I think because a relative path never worked imo

Reply:
done